includes/wikia/parser/templatetypes/TemplateTypesParser.class.php

```diff
@@ -1,8 +1,9 @@
 <?php
 
 class TemplateTypesParser {
+	const CLASS_CONTEXT_LINK = 'portable-context-link';
```

Comment:
Since this is going to be served only on Mercury, I'd suggest removing the portable- prefix